.eslintrc.js

```diff
@@ -319,6 +319,7 @@ module.exports = {
 						'FontSizePicker',
 						'NumberControl',
 						'RangeControl',
+						'TextControl',
```

Comment:
Moved to the stricter rule that doesn't allow `__next40pxDefaultSize={ false }`, now that we're done migrating.

Reply:
Neat!